src/templates/Header/Header.module.scss

```diff
@@ -0,0 +1,8 @@
+.graylinks {
+    align-items: center;
+    display: flex;
+    font-family: 'Roboto Mono', "Helvetica","Arial", sans-serif;
+    font-size: 1rem;
+    gap: 1rem; 
+    letter-spacing: 0.15px;
```

Comment:
`font-familly`, `font-size` and letter-spacing is not needed

for font we should use default font
`font-size: 1rem` doesn't change anything